meson.build

```diff
@@ -0,0 +1,67 @@
+project(
+  'flatpak-builder',
+  'c',
+  license: 'LGPL-2.1-or-later',
+  meson_version: '>= 0.56.2',
+  version: '1.3.1',
+  default_options: 'c_std=gnu99',
+)
+
+cc = meson.get_compiler('c')
+
+add_project_arguments(
+  '-Werror=empty-body',
```

Comment:
I would also maybe wrap this in `cc.get_supported_arguments()`. Makes it a little easier to support clang.

Reply:
Done